views/assets/src/tailwind.css

```
@@ -2,6 +2,23 @@
 @tailwind components;
 @tailwind utilities;
 
+/* Inter — self-hosted variable font (OFL) */
+@font-face {
+  font-family: 'Inter';
```

Stylelint (17.14.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)